server/server/datasources.json

```diff
@@ -2,5 +2,25 @@
   "db": {
     "name": "db",
     "connector": "memory"
+  },
+  "UniteMySQL": {
```

Comment:
Is this the right approach - to commit this information into the repo ? It should be set in environment vars ?

Reply:
changes were done please verify it